spec/lib/data_update_scripts/add_work_profile_field_spec.rb

```diff
@@ -4,6 +4,8 @@
 )
 
 describe DataUpdateScripts::AddWorkProfileField do
+  before { ProfileField.destroy_by(label: "Work") }
```

Comment:
Making sure the field doesn't exist as we seed all profile fields from CSV in `rails_helper.rb`:

```ruby
    # NOTE: @citizen428 needed while we delegate from User to Profile to keep
    # spec changes limited for the time being.
    csv = Rails.root.join("lib/data/dev_profile_fields.csv")
    ProfileFields::ImportFromCsv.call(csv)
    Profile.refresh_attributes!
```

This has been around for a while and we have to see what to do with it long-term.